js/quote-generator.js

```diff
@@ -0,0 +1,55 @@
+const randomBtn = document.querySelector("#random-btn");
+const allAuthorsBtn = document.querySelector("#all-authors-btn");
+const quote = document.querySelector(".quote");
+let quoteAuthor = document.querySelector(".quote-author");
+let quoteGenre = document.querySelector(".quote-genre");
+const allAuthorsContainer = document.querySelector(".all-authors-container");
+const gridContainer = document.querySelector(".grid-container");
+const topButtons = document.querySelector(".top-buttons");
+const footer = document.querySelector("footer");
+
+async function getRandomQuote() {
+  const promise = await fetch("https://api.quotable.io/random");
```

Comment:
HTTP requests are very expensive operations in the world of web. Network of your user can be very slow, so you need provide a better UI for showing your user that he needs to wait. 

It can be either a loader, or [skeleton](https://www.uxdesigninstitute.com/blog/whats-a-skeleton-screen/)